setup.py

```diff
@@ -14,7 +14,7 @@
 __version__ = version["__version__"]
 
 install_requires = [
-    "openai>=1.3,<1.21",
+    "openai>=1.3",
```

Comment:
Remove this change

Reply:
I have removed this change.

Reply:
I don't see it yet.

Reply:
Apologies, was having difficulty pushing for a few minutes. It should be there now.